src/js/post-911-gib-status/components/UserInfoSection.jsx

```diff
@@ -25,6 +25,42 @@ class UserInfoSection extends React.Component {
       );
     }
 
+    let entitlementInfo;
+    if (enrollmentData.percentageBenefit === 0 && enrollmentData.originalEntitlement === 0) {
```

Comment:
This leads me to a more general question of where is the right place to be storing this logic. At first I thought all the logic should live in `Main` or in helpers, and then values like `currentlyAllowed` are just passed to the components as props. That might be the cleanest way to do this, but if we decide to go that route we can refactor later.